manage-images.md

```diff
@@ -0,0 +1,120 @@
+
+Each image gets a unique ID assigned which can be used to retrieve further information about the image with `amc image show <ID>`:
+
+```bash
+id: bcpm2r2hmss0427lkrn0
+name: default
+status: active
+versions:
+    0:
+        size: 374.69MB
+        created-at: 2018-06-27 10:05:32 +0000 UTC
+        status: active
+```
+
+Similar to applications managed by AMS (see [Application Management](https://discourse.ubuntu.com/t/managing-applications/17760)), images have a list of versions too. A new version is created each time an image is updated.
+
+## Default image
+When adding your first image to AMS, it is marked as the default image.
+The default image is used when you create an application without the `image` field or launch a raw container without specifying any ID.
+
+You can set any image as your default with the following command:
+
+    amc image switch bcpm2r2hmss0427lkrn0
+
+Running `amc image list` will now show this image marked as default.
+
+> **Hint:** Note that you cannot delete an image marked as default. You'll need to set a new default image first. *(does not apply if you only have one image left)*
+
+## Update an existing image
+
+As each release of Anbox Cloud comes with a new image which includes various updates to the Ubuntu or Android systems, AMS allows to easily update an existing image with a new version.
+
+Use the following command to update an image in AMS:
+
+    amc image update default anbox-lxd-bionic_1.7_amd64.tar.xz
+
+`default` is the name assigned to the existing image. Uploading the new image to the connected LXD cluster will take a moment. Once the upload has finished, the image is marked as `active`.
+
+Now that a new image version is available to AMS, it will start to create new versions of all available applications but won't publish them. Creating new versions of each application can take a moment depending on the current load of the cluster. You can see available versions of a specific application with the following command:
+
+    amc application show <id>
+
+Each version will give information about which version of an image it is based on.
+
+## Delete an image
+
+Deleting an image will make it unavailable to Anbox Cloud. However, it will not affect any application based on the image directly, as the application keeps a copy of the image internally. As the image is not available to AMS anymore after deleting it, updating the underlying image of applications with a new version is not possible anymore.
+
+Deleting a specific image can be achieved with the following command, where `default` is the name of the image to delete:
+
+    amc image delete default
+
+Specific image versions can be deleted too, which is useful when all applications were migrated to a newer version and the old version is not needed anymore. The only requirement is that a single version of the image is available at all times.
```

Comment:
The last sentence was "Only requirement is that an image as a single version available at all time." - not sure what that is supposed to mean. ;) Does my update capture it?

Reply:
I think the previous sentence misspelled `has` as `as` so your update is correct :)

Reply:
Oh, that suddenly makes sense. :grin: